src/main/java/htsjdk/variant/vcf/VCFHeaderLineTranslator.java

```diff
@@ -65,10 +73,23 @@
     /**
      * parse a VCF4 line
      * @param valueLine the line
+     * @param expectedTagOrder List of expected tags
      * @return a mapping of the tags parsed out
      */
     @Override
     public Map<String, String> parseLine(String valueLine, List<String> expectedTagOrder) {
+        return parseLine(valueLine, expectedTagOrder, Collections.emptySet());
```

Comment:
This is a good candidate for a default method in `VCFLineParser`. Once you have that, you no longer need a method here.

Reply:
Good idea, sorry still mentally stuck in Java 7 world.